src/components/popover-container/popover-container.style.js

```diff
@@ -86,11 +90,16 @@ PopoverContainerCloseIcon.defaultProps = {
   theme: baseTheme
 };
 
+PopoverContainerOpenIcon.defaultProps = {
+  theme: baseTheme
+};
+
 export {
   PopoverContainerWrapperStyle,
   PopoverContainerIcon,
   PopoverContainerHeaderStyle,
   PopoverContainerContentStyle,
   PopoverContainerCloseIcon,
-  PopoverContainerTitle
+  PopoverContainerTitle,
```

Comment:
```suggestion
  PopoverContainerTitleStyle,
```
The components that simply style a HTMLElement should be called `...Style`